public-ui/src/environments/environment.local.ts

```diff
@@ -9,7 +9,7 @@ export const environment = {
   gtmId: 'GTM-NQ9XDTW',
   gtmAuth: 'kMy-ZUX8XsBfuanMZzP_5A',
   gtmPreview: 'env-25',
-  copeFlag: false,
+  copeFlag: true,
```

Comment:
I forget to change this devel flag each time and find why cope is not showing up so leaving this to be true this time.
You can flip it back if you think leaving it on is a problem.